### ISSUE TITLE
Missing `--rpm-systemd` flag?

### ISSUE THREAD

**Issue by mjulian** [jordansissel/fpm#1163]
There are flags for `--rpm-init`, `--deb-init`, and `--deb-systemd`, but `--rpm-systemd` seems to be missing from this list.

**Comment by jordansissel** [comment]
I'd honestly like to remove all of these in favor of somehow integrating the new `pleaserun` support into this, though I'm not sure how to do it just yet.

I am +1 in favor of adding `--rpm-systemd` until we have the above integration availble.

**Comment by ndobbs** [comment]
I too would like to +1 to include an `--rpm-systemd` option to FPM. I need this functionality and wondered why there was a `--deb-systemd` and nothing for RPM. 

Thanks for all of your work with the project.

**Comment by monokal** [comment]
+1

**Comment by dmjio** [comment]
Hello. Have there been any updates on this? Would be glad to help in any way I can.

**Comment by josegonzalez** [comment]
There aren't any updates, but pull requests are welcome :)

**Comment by unsaturated** [comment]
Your situation may vary, however, the following is a workaround if you're generating a package from a directory:

1. Convert your directory to a deb package and use `--deb-systemd`
2. Convert that deb package to rpm (e.g. `fpm -s deb -t rpm *.deb`)
3. Inspect the rpm and you should see the `.service` file in `/lib/systemd/system`

Thank you to all the contributors on fpm. Great work!

**Comment by mohd-akram** [comment]
I've opened pull request #1405 for this.

**Comment by s1na** [comment]
@jordansissel Could you please consider keeping the `init` and `systemd` flags (along with merging [#1045](https://github.com/jordansissel/fpm/pull/1405)), instead of deprecating them in favor of `pleaserun`?

This way it would be possible to use custom systemd units or sysv services, which is helpful when using FPM to ship a multi-platform package.

**Comment by jordansissel** [comment]
I am OK with your proposal to keep the flags.

**Comment by perfecto25** [comment]
for RPM pkg workaround, 

1. place your systemd startup script into the dir that you will be packaging as RPM,
2. run FPM with -s dir, provide a --after-install script that will move your systemd startup from the install dir to /etc/systemd/system
3. when you install the RPM, the after install script moves the system script over to the right location,

for example, 

packaging scriipt,

```
echo "....CREATING Bitbucket RPM from stage dir"
/usr/local/bin/fpm -s dir -t rpm \
**--after-install additionals/after-install** \
-C stage/ \
--name $app \
--version ${version} \
--iteration "1" \
--workdir $(pwd) \
--architecture x86_64 \
--maintainer "Atlassian" \
--description "Atlassian $app" \
--url "www.atlassian.com"  \
--package $(pwd)/RPM/${app}-${version}-1.x86_64.rpm \
-d $db_name

```

"after-install" script
```
# move systemd startup
mv /opt/atlassian/$app/$app.service /etc/systemd/system/
systemctl enable $app.service

```

its hacky but it works.

**Comment by mohd-akram** [comment]
@jordansissel As you plan to keep the flags, would you consider merging my PR for `--rpm-systemd`?

**Comment by jordansissel** [comment]
@mohd-akram yes indeed! I will follow up with you on https://github.com/jordansissel/fpm/pull/1405

**Comment by ericrange** [comment]
why not merged yet? i need this!!! :(

**Comment by Varriount** [comment]
@ericrange In the meantime, you can (ab)use the `--config-files` option to mark a systemd unit file as a config file, and include. The unit file can either be in a relative path mirroring it's root destination (`./etc/systemd/system/file` or `/etc/systemd/system/file`.

**Comment by hertznsk** [comment]
Unexpectedly find out that the `--rpm-systemd` flag is missing. Is there any news about this?

**Comment by zelch** [comment]
I came here because of this.

And now that it's 2025, the pleaserun integration seems far less useful, because almost nothing is shipping with anything except systemd these days.
